examples/simulation/Tutorial1/macros/run_tutorial1.C

```diff
@@ -160,8 +160,11 @@ int run_tutorial1(int nEvents = 10, string mcEngine = "TGeant3", bool isMT = tru
 
     // ------------------------------------------------------------------------
 
+    auto wasMT = run->WasMT();
+    run.reset();
```

Comment:
So good to see unique_ptr working for you!

```suggestion
    auto wasMT = run->WasMT();

    // Destruct and de-allocate the FairRunSim.
    run.reset();
```

I was actually looking for a `reset` method on FairRunSim and then realized that `run` is a unique_ptr… so maybe put a comment here?